package.json

```diff
@@ -16,6 +16,7 @@
         "npm": "^3.0.0"
     },
     "scripts": {
+        "install": "node scripts/fix-modules.js",
```

Comment:
It needs to be `postinstall`

Reply:
It's the same according to the npm official documentation AFAIK.

Reply:
Fixed